style.css

```diff
@@ -134,3 +329,172 @@ img {
     gap: 20px;
   }
 }
+
+/* about */
+.about {
+  width: 100%;
+  height: 100%;
+  background-color: var(--color-background-black);
+  padding: 80px 80px;
```

Comment:
figmaに倣って値を変えてください

```suggestion
  padding: 80px 0;
```

![スクリーンショット 2025-04-15 1 36 14](https://github.com/user-attachments/assets/ade85ebf-8671-4180-82d7-548de696e6be)
![スクリーンショット 2025-04-15 1 36 34](https://github.com/user-attachments/assets/43a31648-499f-447d-b354-f58f08e9982d)